### ISSUE TITLE
Replace aria attribute with alt text for workplace wordmark

### ISSUE THREAD

**Issue by brentryanjohnson** [GSA/wp2030-microsite-new#45]
- I think the workplace wordmark qualifies as page content (not just decoration), and consequently should have associated alt text

**Comment by brentryanjohnson** [comment]
This change is complete and merged.
